.github/workflows/docker-unified.yml

```diff
@@ -314,6 +314,8 @@ jobs:
         run: |
           ./gradlew :datahub-frontend:dist -x test -x yarnTest -x yarnLint --parallel
           mv ./datahub-frontend/build/distributions/datahub-frontend-*.zip datahub-frontend.zip
+        env:
+          NODE_OPTIONS: "--max-old-space-size=3072"
```

Comment:
why is this change necessary?

Reply:
Oh it's to resolve the failing build that was caused by `FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory.` in https://github.com/datahub-project/datahub/actions/runs/3804191769/jobs/6471221050. This line simply increases the memory limit allocated to node. If there are other solutions please feel free to make changes to the PR too!

Reply:
it looks like this failure is still happening :( do you know if we need to put this type of change elsewhere?

Reply:
another initiative we've discussed is removing the unneeded dependencies from our package.json - unsure how many things we can remove but if we can shrink our dependencies this may help solve this problem! it's something i'm looking into and will see if it helps here

Reply:
Hmm, the latest build passed though: https://github.com/datahub-project/datahub/actions/runs/3805789729/jobs/6559405831. The only pipeline that's failing is https://github.com/datahub-project/datahub/actions/runs/3805789729/jobs/6559407548 which looks unrelated to the documentation, probably a flaky test (but I can't rereun it :x)

Reply:
Also let me know if you need help looking into shrinking the dependencies 🗡️ !

Reply:
ah so it looks like it's fixed in the Docker, Build, Scan, Test runs, but i'm still seeing the error in the logs on the general build & test / build job (https://github.com/datahub-project/datahub/actions/runs/3805789734/jobs/6521531814) - you'll have to search for it cuz that's a huge log lol

also it seems that that's timing out after 60 minutes, my hunch is that it's related to this build failing or even that something is taking much longer for some reason.